generators/client/templates/src/main/webapp/app/blocks/config/_http.config.js

```diff
@@ -8,11 +8,6 @@
     httpConfig.$inject = ['$urlRouterProvider', '$httpProvider', 'httpRequestInterceptorCacheBusterProvider', '$urlMatcherFactoryProvider'];
 
     function httpConfig($urlRouterProvider, $httpProvider, httpRequestInterceptorCacheBusterProvider, $urlMatcherFactoryProvider) {
-        <% if (authenticationType == 'session') { %>
```

Comment:
No, it's not required now. Before JHipster explicitly overrided angular defaults. By default angular always use XSRF protection (if you provide right cookie)